packages/aws-cdk/test/integ/helpers/cdk.ts

```diff
@@ -367,7 +511,9 @@ let sanityChecked: boolean | undefined;
 async function ensureBootstrapped(fixture: TestFixture) {
   // Old-style bootstrap stack with default name
   if (await fixture.aws.stackStatus('CDKToolkit') === undefined) {
-    await fixture.cdk(['bootstrap', `aws://${await fixture.aws.account()}/${fixture.aws.region}`]);
+    await fixture.cdkBootstrapLegacy({
```

Comment:
Yeah see this I'm not much of a fan of.

Feels this should just be "default" bootstrap, no?

Reply:
Reveerted.